asset_dashboard/static/js/PortfolioPlanner.js

```diff
@@ -298,10 +306,70 @@ class PortfolioPlanner extends React.Component {
   calculateTotals(portfolio) {
     return {
       budgetImpact: portfolio.reduce((total, project) => { return total + project.budget }, 0),
-      projectNames: portfolio.map(project => project.name),
-      projectZones: portfolio.map(project => project.zones.split(','))
+      totalEstimatedCostByYear: this.calculateEstimatedCostByKey(portfolio, 'year', 'total_estimated_cost'),
+      totalFundedAmountByYear: this.calculateFundedAmountByYear(portfolio),
+      totalEstimatedZoneCostByYear: this.calculateZoneCostByYear(portfolio)
```

Comment:
these are calculated on the fly as projects are added/removed from the portfolio. it uses numbers calculated on the backend. would be nice to do it all on the backend but that would require a more significant change.